Changes

```diff
@@ -3,6 +3,9 @@ Revision history for Data-Validate-Sanctions
 {{$NEXT}}
         last_updated will now return the timestamp of the latest list that was updated if no argument is provided
 
+0.12  2016-11-15 11:44:58 CST
+        - Improving the search for larger sanction lists
+
```

Comment:
What current version is it on cpan.org ? IMHO it should be version 0.13 for this change:
```suggestion
{{$NEXT}}
        - Improving the search for larger sanction lists
        
0.12  2016-11-15 11:44:58 CST
        - last_updated will now return the timestamp of the latest list that was updated if no argument is provided


```

Reply:
fixed, thanks